src/tagging/tests/category.modifier.test.js

```diff
@@ -1,24 +1,25 @@
 import React from 'react';
-import CategoryModifier from '../components/categoryModifier';
+import CategoryModifier from '../components/InnerComponents/CategoryModifier';
 import renderer from 'react-test-renderer';
-import { shallow } from 'enzyme';
 
-const selectedTagCategory = { description: 'animal', id: 1};
+const selectedTagCategory = { description: 'animal', id: 1 };
 const onTagCategoryChange = jest.fn();
 const tagCategories = [
   { description: 'Name', id: 1 },
   { description: 'Number', id: 2 },
   { description: 'Animal', id: 3 },
-  { description: 'Food', id: 4 },
+  { description: 'Food', id: 4 }
 ];
 
 describe('TagCategory Component', () => {
   it('match snapshot', () => {
-    const component = renderer.create(<CategoryModifier
-      selectedTagCategory={selectedTagCategory}
-      onTagCategoryChange={onTagCategoryChange}
-      tagCategories={tagCategories}
-    />);
+    const component = renderer.create(
```

Comment:
Please change this to enyzme, reactRenderer will be removed in future. Examples in #39